certified-connectors/Leadinfo/apiDefinition.swagger.json

```diff
@@ -0,0 +1,293 @@
+              "type": "string",
+              "description": "vat_number"
+            },
+            "employees": {
+              "type": "integer",
+              "format": "int32",
+              "description": "employees"
+            },
+            "employees_total": {
+              "type": "string",
+              "description": "employees_total"
+            },
+            "tags": {
+              "type": "string",
+              "description": "tags"
+            },
+            "sales_volume": {
+              "type": "string",
+              "description": "sales_volume"
+            },
+            "comment": {
+              "type": "string",
+              "description": "comment"
+            },
+            "sales_volume_raw": {
+              "type": "integer",
+              "format": "int32",
+              "description": "sales_volume_raw"
+            },
+            "sales_volume_symbol": {
+              "type": "string",
+              "description": "sales_volume_symbol"
+            },
+            "website_url": {
+              "type": "string",
+              "description": "website_url"
+            },
+            "inbox_url": {
+              "type": "string",
+              "description": "inbox_url"
+            },
+            "legal_form_name": {
+              "type": "string",
+              "description": "legal_form_name"
+            },
+            "founding_year": {
+              "type": "string",
+              "description": "founding_year"
+            },
+            "branch_name": {
+              "type": "string",
+              "description": "branch_name"
+            },
+            "branch_local_01_name": {
+              "type": "string",
+              "description": "branch_local_01_name"
+            },
+            "branch_international_01_name": {
+              "type": "string",
+              "description": "branch_international_01_name"
+            },
+            "linkedin": {
+              "type": "string",
+              "description": "linkedin"
+            },
+            "xing": {
+              "type": "string",
+              "description": "xing"
+            },
+            "leadinfo_link": {
+              "type": "string",
+              "description": "leadinfo_link"
+            },
+            "timestamp": {
+              "type": "string",
+              "description": "timestamp"
+            },
+            "page_url_first_visit": {
+              "type": "string",
+              "description": "page_url_first_visit"
+            },
+            "branch_code": {
+              "type": "string",
+              "description": "branch_code"
+            },
+            "branch_code_international": {
+              "type": "string",
+              "description": "branch_code_international"
+            }
+          }
+        },
+        "description": "webhook response"
+      },
+      "put": {
+        "summary": "Triggers when a company is send",
```

Comment:
@GerbenZ01 all files should be free of grammatical errors and typos. I think here meant to say "Triggers when a company is sent"